ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
@calbach Thanks, that's very helpful.

One question I have, however, is about `readNumber`.  The SAM specification says (in 1.4.2): "If 0x40 and 0x80 are both set, the read is part of a linear template, but it is neither the first nor
the last read. If both 0x40 and 0x80 are unset, the index of the read in the template is unknown.
This may happen for a non-linear template or the index is lost in data processing."

So it seems like 0x40 and 0x80 either both being set or both not being set are valid states.  However, `readNumber` seems to capture only the two other states, namely when one and only one of either 0x40 or 0x80 is set.  Is there a way in the GA4GH spec right now to capture the both-set or both-unset states?